tools/perf/src/main/java/org/apache/bookkeeper/tools/perf/journal/JournalWriter.java

```diff
@@ -272,7 +272,7 @@ void execute(Journal[] journals) throws Exception {
             for (Journal journal : journals) {
                 Checkpoint cp = journal.newCheckpoint();
                 try {
-                    journal.checkpointComplete(cp, true);
+                    journal.checkpointComplete(cp, true, null);
```

Comment:
what about adding a comment here ?
`null` means that the checkpoint is not started by a specific LedgersDirManager

Reply:
I have added the javadoc on the `checkpointComplete` method interface.